src/main/webapp/app/entities/quiz-exercise/quiz-exercise.state.js

```diff
@@ -13,7 +13,7 @@
             parent: 'entity',
             url: '/quiz-exercise',
             data: {
-                authorities: ['ROLE_USER'],
+                authorities: ['ROLE_ADMIN', 'ROLE_TA'],
```

Comment:
same as above.. I am note sure what this js function is supposed to do, but are you sure that normal users should not be able to do it?

Reply:
This state is for listing all quiz-exercises in the "Entities" View. Normal users shouldn't have access to that part anyway.